scripts/qesap/test/unit/test_qesap_lib_cmd_ansible.py

```diff
@@ -0,0 +1,37 @@
+from unittest import mock
+import logging
+import os
+
+
+from lib.cmds import export_ansible_output
+
+log = logging.getLogger(__name__)
+
+
+def test_export_ansible_output():
+    '''
+    This test checks if the ansible execution generates log files
+    in this case generates the generic one "ansible.testAll.log.txt"
+    scripts/qesap/test/unit/test_qesap_ansible.py
+    
+    '''
+
+    test_dir = os.getcwd()
+    test_file = os.path.join(test_dir, 'ansible.testAll.log.txt')
+    command_to_sent = {'test1':'test_value1', 'test2':'test_value2', 'test3':'test_value3', 'test4':'test_value4',
+                        'cmd':['/tmp/exec_venv/bin/ansible', '-vvvv', '-i', '/root/qe-sap-deployment/terraform/aws/inventory.yaml',
+                              'testAll', '-a', '--ssh-extra-args="-l cloudadmin -o UpdateHostKeys=yes -o StrictHostKeyChecking=accept-new"']}
+    ansible_output = """DEBUG    OUTPUT: ansible [core 2.13.5]
```

Comment:
does no matter at all for `export_ansible_output` what is the content of this string, but generally speaking content will not be like this but 

```
ansible_output = """ansible [core 2.13.5]
                        config file = None
                        configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
                        ansible python module location = /tmp/exec_venv/lib64/python3.11/site-packages/ansible
                        ansible collection location = /root/.ansible/collections:/usr/share/ansible/collections
                        executable location = /tmp/exec_venv/bin/ansible
                        python version = 3.11.5 (main, Sep 06 2023, 11:21:05) [GCC]
                        jinja version = 3.1.4
                        libyaml = True"""
```

Reply:
yes indeed doesn't matter